app/Http/Controllers/Auth/AdmissionController.php

```diff
@@ -0,0 +1,223 @@
+    }
+
+    /**
+     * @param Request $request
+     * @return View
+     * @throws AuthorizationException
+     */
+    public function index(Request $request): View
+    {
+        $authUser = $request->user();
+        $this->authorize('viewSome', Application::class);
+
+        $applications = Application::query();
+        $filtered_workshop = $this->getFilteredWorkshop($request);
+        $accessible_workshops = $this->getAccessibleWorkshops($authUser);
+
+        $applications->join('workshop_users', 'applications.user_id', '=', 'workshop_users.user_id');
+        $applications->whereIn('workshop_id', $accessible_workshops->pluck('id'));
+        if ($filtered_workshop) {
+            $applications->where('workshop_id', $filtered_workshop->id);
+        }
+
+        //hide unfinished
+        if ($authUser->cannot('viewUnfinished', Application::class)) {
+            $applications->where('submitted', true);
+        }
+        //filter by status
+        //        if ($request->has('status')) {
+        //            $applications->where('status', $request->input('status'));
+        //        }
+        return view('auth.admission.index', [
+            'applications' => $applications->with('user.educationalInformation')->get()->unique()->sortBy('user.name'),
+            'workshop' => $request->input('workshop'), //filtered workshop
+            'workshops' => $accessible_workshops, //workshops that can be chosen to filter
+            'status' => $request->input('status'), //filtered status
+            'applicationDeadline' => $this->getDeadline(),
+            'periodicEvent' => $this->periodicEvent()
+        ]);
+    }
+
+    /**
+     * Show and application's details
+     *
+     * @param Application $application
+     * @return View
+     * @throws AuthorizationException
+     */
+    public function show(Application $application): View
+    {
+        $this->authorize('view', $application);
+        $user = User::withoutGlobalScope('verified')->with('application')->find($application->user_id);
+        return view('auth.admission.application', ['user' => $user]);
+    }
+
+    /**
+     * Edit an application's note.
+     * @param Request $request
+     * @return RedirectResponse
+     * @throws AuthorizationException
+     */
+    public function update(Request $request, Application $application): RedirectResponse
+    {
+        $this->authorize('view', $application);
+        $newStatus = $request->input('status_'.$application->user->id);
+        if ($request->has('note')) {
+            $application->update(['note' => $request->input('note')]);
+        } elseif ($newStatus) {
+            $application->update(['status' => $newStatus]);
+        }
+        return redirect()->back();
+    }
+
+    /**
+     * Accept and delete applciations.
+     * @return RedirectResponse
+     * @throws AuthorizationException
+     */
+    public function finalize(): RedirectResponse
```

Comment:
How do we plan to do this? I don't know how we should differentiate between admitted and banished applicants if there are no statuses anymore.

Reply:
that will come later in a future PR. We have the statuses in https://github.com/EotvosCollegium/mars/pull/574